content/trial_getting_started/examples/openstack_hello_world_example.md

```diff
@@ -7,6 +7,172 @@ alwaysopen = false
 
 {{%children style="h2" description="true"%}}
 
-## Step 1: Install the Cloudify Manager as a Docker container
 
-The easiest way to deploy Cloudify trial manager is as a Docker container. This tutorial assumes that you have [Docker](https://docs.docker.com/install) installed on your local machine.
+This Example demonstrates a simple topology setup in **Openstack**, the deployment consists of a VM, a simple web service + app, and all of the essential peripherals in Openstack (security group, network, etc.)
+
+Cloudify allows for multiple user interfaces. In this tutorial we will demonstrate the usage of the Cloudify management console (web UI) and the Cloudify command line (cfy). The following steps demonstrate both approaches.
+
+## Step 1: Install the Cloudify Manager inside Docker container
+
+In order to deploy the Cloudify manager inside Docker container follow the instructions in [this page]({{< relref "trial_getting_started/trial_install.md" >}}).
+
+## Step 2: Create the secrets containing Openstack credentials
+
+To connect to Openstack a set of credentials are required. Cloudify recommends storing such sensitive information in a Cloudify secret. Secrets are kept encrypted in a secure way and used in run-time by the system. learn more about Cloudify secrets [here]({{< relref "/cli/orch_cli/secrets.md" >}}).
+
+Store the secrets in the manager:
+
+This can be done through command line or directly via Cloudify management console.
+
```

Comment:
I think if we tell the user to `source` the Openstack RC file, we can then tell them to use the environment variables, for example:

```
docker exec -it cfy_manager_local sh -c   "cfy secrets create openstack_username -s ${OS_USERNAME}"
```

Reply:
This could simplify the copy-pasting experience for Openstack users.

Reply:
Good direction, How do I "source" the Openstack RC file? Also, we should probably guide them how to get this file.

Reply:
in racspace its different from memset so i think  an explanation will not give us value here

Reply:
https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux_OpenStack_Platform/4/html/End_User_Guide/cli_openrc.html